.eslintrc.js

```diff
@@ -259,6 +259,7 @@ module.exports = {
           // before `import styles from "styles.module.css"`, which is what we
           // always did. This should be removable once we stop using ambient
           // module declarations for theme aliases.
+          {pattern: '@docusaurus/**', group: 'internal'},
```

Comment:
@MarkShawn2020 I'm wondering why you think the order before was not correct. I wrote this config myself and I've spent hours until I'm sure all the current conventions are enforced. Now ESLint is giving dozens of warnings when you run `yarn lint`.

Please assume whatever ESLint is telling you is correct, until maintainers have explicitly told you otherwise. That's the point of a linter.

Reply:
fine, I would investigate it after resolving snapshot tests.

Reply:
You would just need to revert https://github.com/facebook/docusaurus/pull/7849/commits/2649a4e6dcef09e7c354bd9a1bb1647bdb7dc24a. The code before was correct—we always put CSS import last, even after type imports.

Reply:
Not enough, since I initialized previous `I18nLocaleConfig` with an explicit key of `charMap`, now it failed to pass the snapshot test.

Reply:
I was talking about the lint errors—the snapshot errors can be fixed by just re-running `yarn test -u`.

Reply:
hmmm.... I reverted, but also merged with a new snapshot, do I actually need to delete them then ?